demos/sample_azure_iot_pnp/sample_azure_iot_pnp.c

```diff
@@ -0,0 +1,1035 @@
+/* Copyright (c) Microsoft Corporation. All rights reserved. */
+/* SPDX-License-Identifier: MIT */
+
+/* Standard includes. */
+#include <string.h>
+#include <stdio.h>
+#include <time.h>
```

Comment:
this looks to be wrong dependency, as we depend upon getunittime api to get time, and that api is provided by sample project ports .... if true for other sample, can we remove from there also.

Reply:
needed for this guy here
https://github.com/Azure-Samples/iot-middleware-freertos-samples/pull/28/files/708592629b6b5f3bc9f2faff509c6f44161ec066#diff-b44ce38db1c6040a3fd014270977865367a1222b4ae0c415c6c5fcdab47de58aR301

Reply:
is it possible to remove this depenency? as we have unix time function provided by port?

Reply:
The problem is the need for all the time components. So not just `time()` but `strftime()` as well. There is a "FreeRTOS Plus Posix" library but we would then need to integrate that which is a labs project:
https://freertos.org/FreeRTOS-Plus/FreeRTOS_Plus_POSIX/index.html